### PR TITLE
a simple parfor example

### PR DESCRIPTION
**Pull request by mpiercy**
a simple parfor example...and well tested

### CODE REVIEW

File: src/docs/software/using/matlab.md [hunk 1]
```diff
@@ -171,6 +171,72 @@ You can check the status of the job with the [`squeue`][url_squeue] command,
 and check the contents of the `matlab_test.JOBID.{out,err}` files to see the
 results.
 
+#### Using multiple CPUs with MATLAB's parfor loop
+
+You can run your MATLAB code across multiple CPUs on Sherlock using [parfor][url_parfor] loops.  Users can run on as many CPUs that are on a node in a single job.  The key is to grab the [SLURM environmental variable][url_SLURM_ENV] ``$SLURM_CPUS_PER_TASK`` and create the worker pool in your MATLAB code with:
+
+```
```
Comment (vsoch) [inline src/docs/software/using/matlab.md:178]:
```suggestion
```matlab
```

File: src/docs/software/using/matlab.md [hunk 2]
```diff
@@ -171,6 +171,72 @@ You can check the status of the job with the [`squeue`][url_squeue] command,
...
+parpool('local', str2num(getenv('SLURM_CPUS_PER_TASK')))
+```
+Here is an sbatch that will use 16 CPUs on a node at once to run a simple MATLAB parfor enabled script.
+
+```
```
Comment (vsoch) [inline src/docs/software/using/matlab.md:183]:
```suggestion
```bash
```

File: src/docs/software/using/matlab.md [hunk 3]
```diff
@@ -171,6 +171,72 @@ You can check the status of the job with the [`squeue`][url_squeue] command,
...
+#SBATCH --mail-type=ALL
+module load matlab
+matlab -nosplash -nodesktop -r "pfor"
+```
+
+Save as parfor.sbatch
+
+MATLAB code:
+
+```
```
Comment (vsoch) [inline src/docs/software/using/matlab.md:200]:
```suggestion
```matlab
```

File: src/docs/software/using/matlab.md [hunk 4]
```diff
@@ -171,6 +171,72 @@ You can check the status of the job with the [`squeue`][url_squeue] command,
...
+fprintf('The parallel Monte-Carlo method is executed in %8.2f seconds.n', T);
+delete(gcp);
+exit;
+
+```
+
+Save as pfor.m
+
+Submit the job:
+```
```
Comment (vsoch) [inline src/docs/software/using/matlab.md:233]:
```suggestion
```bash
```

**Comment by vsoch** [comment]
Actually I'm not sure matlab is supported! https://squidfunk.github.io/mkdocs-material/extensions/codehilite/

**Comment by mpiercy** [comment]
kcgthb,
is there anything holding this up?  Anything else I need to do?

**Comment by kcgthb** [comment]
Sorry I haven't got a chance to look at this yet. Will do!

**Pull request merged**